demo/server.js

```diff
@@ -150,4 +150,52 @@ initSqlJs().then(SQL => {
   });
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
+
+   // ACCURACY RECORD MANAGEMENT
+   function presistAccuracyRecord(accuracy) {
+    db.run('INSERT INTO accuracy VALUES (?, ?, ?, ?, CURRENT_TIMESTAMP);', [
+      accuracy.transactionHash,
+      accuracy.blockNumber,
+      accuracy.modelId,
+      accuracy.accuracy,
+    ]);
+    fs.writeFile(dbPath, Buffer.from(db.export()), () => {});
+  }
+  
+ // Add a new accuracy record for a model
```

Comment:
There's a few styling inconsistencies that I didn't want to buy you about changing. Your editor would handle most of these with automatic formatting. Instead of bugging you to make those changes, I set up some tools for automatic linting. #127 adds support for `yarn lint-fix` so once you pull in main, then you should run `yarn lint-fix` and it should resolve most issues automatically or tell you where there is errors. Then I can finally merge this!

Reply:
Oh I commented on this line because there's an issue with it: 1 space at the beginning instead of 2. Yeah it's a small thing but keeping the style clean and consistent is important for easily maintaining the code. Anyway, the automatic linting should resolve this issue.

Reply:
Okay , i change it now !

Reply:
Hey , used lint fix it fixed all the things itself , you can check now